mgmtsystem_nonconformity_hr/README.rst

```diff
@@ -0,0 +1,108 @@
+=============================
+Management System Extend - HR
```

Comment:
Please change the name here

```suggestion
Management System - Nonconformity HR
```

And also on the manifest

Reply:
Damn, I forgot to do it, sorry. All PR alligned.